src/rulesets/operations.ts

```diff
@@ -47,6 +47,15 @@ export const rules = {
       },
     );
   },
+  operationIdSet: ({ operations }: SnykApiCheckDsl) => {
+    operations.requirement.must(
+      "have operationId",
+      (operation, context, docs) => {
+        docs.includeDocsLink(links.standards.operationIds);
+        expect(operation.operationId).to.be.ok;
+      },
+    );
+  },
```

Comment:
require an operationId, always
compliments rule that checks if it's correct on change